Library/Formula/libsass.rb

```diff
@@ -18,15 +18,11 @@ class Libsass < Formula
 
   def install
     ENV.cxx11
-    ENV["LIBSASS_VERSION"] = "HEAD" if build.head?
+    ENV["LIBSASS_VERSION"] = "3.2.0"
```

Comment:
I admit to not knowing what a head build is. 

The pace of Libsass is fast enough that I imagine `"HEAD"` as a version string would generally be misinformation.